kanban-all/kanban-data/src/main/java/ru/ifmo/p3411/repositories/TaskRepository.java

```diff
@@ -0,0 +1,14 @@
+package ru.ifmo.p3411.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.stereotype.Repository;
+import ru.ifmo.p3411.data.Task;
+
+/**
+ * @author Ellepsis
+ * @since 1.0
+ */
+@Repository
+public interface TaskRepository extends JpaRepository<Task, Long>, JpaSpecificationExecutor<Task> {
```

Comment:
I also suggest you to autowire all of the repositories inside SmokeTest. Just to be sure they're all instantiated and wired successfully.